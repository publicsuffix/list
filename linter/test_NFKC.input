// test:
// - label contains non-NFKC character(s)
//
// best viewed with 'LC_ALL=C.UTF-8 vi <filename>' (or any other UTF-8 locale)

// ===BEGIN ICANN DOMAINS===

südtirol.it
südtirol.it

// ===END ICANN DOMAINS===
