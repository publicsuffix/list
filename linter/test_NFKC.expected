9: error: Rule must be NFKC: 'suÌˆdtirol.it'
11: warning: No PRIVATE section found
